src/core/MIGRATION_EXAMPLES.md

```diff
@@ -466,3 +466,52 @@ elsewhere.
 In most cases, the most convenient way to access these APIs will be via the
 [AppMountContext](/docs/development/core/public/kibana-plugin-public.appmountcontext.md)
 object passed to your application when your app is mounted on the page.
+
+## Render HTML Content
+
+You can return a blank HTML page bootstrapped with the core application bundle from an HTTP route handler
```

Comment:
The document doesn't cover the main purpose of https://github.com/elastic/kibana/issues/41981 - chromeless apps.
Plugin authors don't know why they need to render an HTML page.
They want to know:
- what is a chromeless app?
- what has changed for the `hidden` apps?
- what is the LP API equivalent provided?
- how to make a regular or chromeless app accessible?

Reply:
@restrry nice ideas! A few questions:

> how to make a regular or chromeless app accessible?
> what is the equivalent of LP API in the new platform?

Can you elaborate on what you mean by these?

Reply:
> what is the equivalent of LP API in the new platform?

The legacy platform provides API to register `hidden apps`. We should describe what methods provided instead. Example:  https://github.com/elastic/kibana/pull/54893/files#diff-ca730ff688cc29f7cfb77853005355a1

> how to make a regular or chromeless app accessible?

how to register regular app, how to register chromeless app.